emulsify_twig.info.yml

```diff
@@ -1,6 +1,5 @@
 name: Emulsify Twig Extensions Module
 type: module
 description: BEM and Add Attributes Twig Extensions for Drupal
-core_version_requirement: ^8 || ^9
-version: 2.0.0
+core_version_requirement: ^9 || ^10
```

Comment:
@marcb4k I suspect that the deprecations you fixed aren't actually able to work in all versions of Drupal 9.  The requirement on Drupal 9 is probably 9.3 or 9.4, could even be 9.5 depending on when the new API was introduced.  I think it would be worthwhile to check the deprecations here and bump it up to a later version so if anyone is running 9.2 they don't pull in the D10 compatible code.

Reply:
Looks like the `Twig_Extension` class was deprecated and recommended switching to `AbstractExtension` class was  in Twig 2.7. And seeing as though Drupal 9 upgraded to Twig 2, and seems like [Drupal 9.0.0 upgraded Twig to 2.12.5](https://www.drupal.org/project/drupal/releases/9.0.0) so I think we're good? Let me know your thoughts.